libobs/cmake/os-macos.cmake

```diff
@@ -6,7 +6,8 @@ target_link_libraries(
           "$<LINK_LIBRARY:FRAMEWORK,AudioUnit.framework>"
           "$<LINK_LIBRARY:FRAMEWORK,AppKit.framework>"
           "$<LINK_LIBRARY:FRAMEWORK,IOKit.framework>"
-          "$<LINK_LIBRARY:FRAMEWORK,Carbon.framework>")
+          "$<LINK_LIBRARY:FRAMEWORK,Carbon.framework>"
```

Comment:
This is fine, but what required you to use Carbon (macOS keeps some functionality around in that framework, even though it "shouldn't" be used anymore).

Reply:
Carbon was already there, this just shows a a new line because I moved the parentheses down because I added `Security` at the end (for keychain stuff).